packages/appstore/src/components/account-transfer-modal/account-transfer-modal.tsx

```diff
@@ -24,13 +25,18 @@ const AccountTransferModal = observer(({ is_modal_open, toggleModal }: TAccountT
     const history = useHistory();
 
     React.useEffect(() => {
+        if (is_modal_open) setActiveTab('account_transfer');
```

Comment:
here we set an active container when the user is clicking on Transfer button and modal is opening